services/web-server/src/login/clients/GithubClient.js

```diff
@@ -1,18 +1,50 @@
+const assert = require('assert');
 const request = require('superagent');
 const Debug = require('debug');
 
 const debug = Debug('GithubClient');
 const baseUrl = 'https://api.github.com';
 
 module.exports = class GithubClient {
+  constructor({ accessToken }) {
+    assert(accessToken, 'An access token is required to access Github endpoints');
+
+    this.accessToken = accessToken;
+  }
+
   async userFromUsername(username) {
     const { body } = await request
-      .get(`${baseUrl}/users/${username}`);
+      .get(`${baseUrl}/users/${username}`)
+      .set('Authorization', `Bearer ${this.accessToken}`);
 
     if (!body) {
       debug(`profile for user ${username} not found`);
     }
 
     return body;
   }
+
+  async orgsFromUsername(username) {
+    const { body } = await request
+      .get(`${baseUrl}/users/${username}/orgs`)
+      .set('Authorization', `Bearer ${this.accessToken}`);
```

Comment:
This endpoint only seems to return public org memberships, even with an access token for which I've checked all of the permissions boxes.  I don't know if there's a way around that, but if not we should probably document that clearly: you can only get permissions on an org with public membership.

Reply:
I've switched to using `user/memberships/orgs` which seems to return private org memberships.